frame/system/src/lib.rs

```diff
@@ -1222,8 +1231,8 @@ impl<T: Trait> Module<T> {
 					"WARNING: Referenced account deleted. This is probably a bug."
 				);
 			}
-			Module::<T>::on_killed_account(who.clone());
 		}
+		Module::<T>::on_killed_account(who.clone());
```

Comment:
Whats up with this change?
Its going to trigger this function even if there was no Account entry?

Reply:
This matches to the behavior on create account, which does not actually create Account entry

https://github.com/paritytech/substrate/blob/12b6c5dbe9f7c32ffbd3a3d64a442994f8c279da/frame/system/src/lib.rs#L1260

Otherwise the behavior between local and composite will be different.